pkg/grpc/exec/exec.go

```diff
@@ -232,6 +261,43 @@ func (msg *MsgExecveEventUnix) Cast(o interface{}) notify.Message {
 	return &MsgExecveEventUnix{MsgExecveEventUnix: t}
 }
 
+// finalize() is called to finalize Process of the event ExecveEvent
+//
+// The aim of this function is to finalize events by adding or updating some
+// fields without propagating those into the process cache.
+// These fields could be related to the specific calling event, example only
+// for ProcessExec and not ProcessKprobe.
+//
+// There are two conditions when we could update the event:
+//  1. When the event finalize handler return update set to true, for this a deep copy
+//     of the process was made, and the related fields added then the event.Process
+//     must be set with ev.SetProcess() to this returned copy.
+//  2. The finalize() is called from the event cache retry in this
+//     case new information could have been added, so let's do another
+//     ev.SetProcess() call to update the process of the event.
+func (msg *MsgExecveEventUnix) finalize(ev notify.Event, internal *process.ProcessInternal, cache int) error {
+	// This should never happen by this time
+	if ev.GetProcess() == nil {
+		return process.ErrProcessInfoMissing
+	}
+
+	update := false
+	proc, err := internal.UpdateExecOutsideCache(option.Config.EnableProcessCred, &update)
```

Comment:
Wouldn't be better to have:
```suggestion
	proc, update, err := internal.UpdateExecOutsideCache(option.Config.EnableProcessCred)
```

Reply:
done